ui/src/styles/globals.css

```
@@ -23,7 +23,214 @@
+ * wrapping the story in a themed element.
+ *
+ * This lives in globals.css because every SPA imports it
+ * (`client-admin/src/index.css`) and so does Storybook
+ * (`ui/.storybook/preview.tsx`), and Tailwind processes it in each
+ * consumer — one line covers all of them. Design contract §3.4.1.
+ *
+ * `check-contrast.mjs` compiles a `dark:` candidate and asserts the emitted
+ * rule is attribute-scoped, not media-scoped; grepping this file for the
+ * at-rule would not prove the variant actually reached the bundle.
+ */
+@custom-variant dark (&:where([data-theme="dark"], [data-theme="dark"] *));
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)